src/main/java/chess/Bishop.java

```diff
@@ -1,32 +1,34 @@
 package chess;
-
-import java.util.List;
 import java.util.ArrayList;
 
 class Bishop extends Piece {
-    Bishop(final Color color, final Position source) {
+
+    private DiagonalMovePiece diagonalMovePiece;
```

Comment:
Maybe the new class should be named "DiagonalMovablePiece" and the attribute should be named "diagonalMovablePiece"

Reply:
I have already changed The name of The class. Can you review it, please?